### PR TITLE
documentation: fix MDN url in events.rs

### PR DESCRIPTION
**Pull request by mkroehnert**

### CODE REVIEW

File: src/dom_entity_names/events.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ macro_rules! make_events {
     { $($event_camel:ident => $event:expr),+ } => {
 
         /// The Ev enum restricts element-creation to only valid event names, as defined here:
-        /// [https://developer.mozilla.org/en-US/docs/Web/Evs](https://developer.mozilla.org/en-US/docs/Web/Evs)
+        /// [https://developer.mozilla.org/en-US/docs/Web/Evs](https://developer.mozilla.org/en-US/docs/Web/Events)
```
Comment (MartinKavik) [inline src/dom_entity_names/events.rs]: Please, change also the link description to `[MDN reference]`, thanks

Comment (MartinKavik) [reply src/dom_entity_names/events.rs]: It should also fix linter error.
Run `cargo make verify` to make sure it's ok before pushing.

Comment (mkroehnert) [reply src/dom_entity_names/events.rs]: @MartinKavik would `[MDN reference Web/Events]` also be okay?

Comment (MartinKavik) [reply src/dom_entity_names/events.rs]: yes

Comment (mkroehnert) [reply src/dom_entity_names/events.rs]: @MartinKavik pushed

**Pull request merged**